chapters/classes.tex

```diff
@@ -383,14 +383,9 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\label{
 definition equations is given in \cref{variability-of-expressions}.
 
 \begin{nonnormative}
-A discrete-time variable is a piecewise constant signal which
-changes its values only at event instants during simulation. Such types
-of variables are needed in order that special algorithms, such as the
-algorithm of Pantelides for index reduction, can be applied (it must be
-known that the time derivative of these variables is identical to zero).
-Furthermore, memory requirements can be reduced in the simulation
-environment, if it is known that a component can only change at event
-instants.
+A discrete-time variable is a piecewise constant signal which changes its values only at event instants during simulation.
+There may be additional \lstinline!Real! variables that are piecewise constant without being declared as \lstinline!discrete!.
+Those additional variables may not be used where a \lstininline!discrete! variable is required, but tools may optimize code to only compute them at events.
```

Comment:
I think this is what you mean:
```suggestion
The \lstinline!Real! variables declared as \lstinline!discrete! are a subset of all discrete-time \lstinline!Real! variables.
Only being discrete-time is not sufficient where a \lstinline!discrete! variable is required.
```

Reply:
I understand that it is confusing and tried to clarify.